packages/angular_devkit/build_angular/src/babel/presets/application.ts

```diff
@@ -14,9 +14,26 @@ import type {
   makeLocalePlugin,
 } from '@angular/localize/tools';
 import { strict as assert } from 'assert';
+import browserslist from 'browserslist';
 import * as fs from 'fs';
 import * as path from 'path';
 
+/**
+ * List of browsers which are affected by a WebKit bug where class field
+ * initializers might have incorrect variable scopes.
+ *
+ * See: https://github.com/angular/angular-cli/issues/24355#issuecomment-1333477033
+ * See: https://github.com/WebKit/WebKit/commit/e8788a34b3d5f5b4edd7ff6450b80936bff396f2
+ */
+const safariClassFieldScopeBugBrowsers = new Set(
+  browserslist([
+    // Safari <15 is technically not supported via https://angular.io/guide/browser-support,
+    // but we apply the workaround if forcibly selected.
+    'Safari <=15',
+    'iOS <=15',
```

Comment:
FYI: Discussed with @clydin that we should just apply for v14 too. The range can be that loose just to be safe. older versions will have the plugin enabled anyway.